saleor/core/utils/random_data.py

```diff
@@ -1201,16 +1202,49 @@ def create_shipping_zones():
     )
 
 
+def create_click_and_collect_related_fields() -> List[Tuple[bool, str]]:
```

Comment:
I am not sure of this function. It is super hard to understand what it is doing. 
It looks like it generates some list of tuple which you use randomly. It is used in one place so no need to return a list here. 
why not something like this:
```
def foo() -> Tuple[bool, str]:
    is_private = random.choice([True, False])
    c_and_c_option = random.choice(["WarehouseClickAndCollectOption with excluding local stock"]
    return is_private, c_and_c_option

...
is_private, cc_option = foo()

OR better
is_private = random.choice([True, False])
c_and_c_option = random.choice([Warehouse exlucding local stock])
```

Reply:
Yes, I had have over engineered it, since I want a full collection before entering the loop ;d I have use your second proposal, done.